server/src/service/admin.agenda.ts

```diff
@@ -298,7 +341,11 @@ export const retrieveAll = async (): Promise<schema.AdminAgenda[] | null> => {
 
 export const remind = async ({
   id,
-}: schema.Remind): Promise<schema.Remind[] | null> => {
+}: schema.Remind): Promise<{
+  users: string[];
```

Comment:
변수명은 `users`인데 타입이 `string[]`이면 이것이 user의 정확히 어떤 것인지(id인지, username인지, displayName인지 등) 알기 힘들 것 같아요. `userIds`, `usernames`, `userDisplayNames` 등의 변수명을 사용하는게 조금 더 직관적인 것 같아요. 

이 상황에서는 username의 array를 리턴하는 것이니 `usernames`가 적절해 보이네요.

```suggestion
  usernames: string[];
```